### PR TITLE
Improve histogram

### PR DESCRIPTION
**Pull request by namma**
ddf.getVectorHistogram is now using implementation getVectorHistogramImp inside file BinningHandler.scala

### CODE REVIEW

**Comment by khangich** [comment]
ok to test

**Comment by namma** [comment]
it passed many unit tests, including DDF with missing values and/or invalid data. Please merge.

File: spark/src/main/scala/io/spark/ddf/analytics/BinningHandler.scala [hunk 1]
```diff
@@ -1,18 +1,45 @@
...
 
 class BinningHandler(mDDF: DDF) extends ABinningHandler(mDDF) with IHandleBinning {
 
+
+  def parseDouble(r: Row) = try {r.get(0).toString.toDouble } catch { case _ => None }
+
+  override def getVectorHistogramImpl(columnName: String, numBins: Int): java.util.List[AStatisticsSupporter.HistogramBin] = {
+    val projectedDDF: DDF = mDDF.VIEWS.project(columnName)
+    val rdd: RDD[Row] = projectedDDF.getRepresentationHandler.get(classOf[RDD[_]], classOf[Row]).asInstanceOf[RDD[Row]]
+    val rdd1 = rdd.map(r => {try {r.get(0).toString.toDouble } catch { case _ => None }})
```
Comment (Huandao0812) [inline spark/src/main/scala/io/spark/ddf/analytics/BinningHandler.scala]: bad way to parse double

Comment (namma) [inline spark/src/main/scala/io/spark/ddf/analytics/BinningHandler.scala]: Can you elaborate? This one is to eliminate missing data or invalid value.
What is your suggestion?

Comment (mbbui) [inline spark/src/main/scala/io/spark/ddf/analytics/BinningHandler.scala]: Huan, you should tell why it is bad. If Nam already knows why it is bad he would not do this already.

Comment (namma) [inline spark/src/main/scala/io/spark/ddf/analytics/BinningHandler.scala]: I talked with Huan. He said try-catch significantly decreases the performance. I will check and update the code accordingly when I have cycle. At the moment, I believe it is not a stopper issue though.

**Pull request merged**